services/src/main/java/org/exoplatform/webconferencing/rest/RESTWebConferencingService.java

```diff
@@ -626,4 +633,28 @@ public Response getRoomInfo(@Context UriInfo uriInfo,
                      .build();
     }
   }
+
+  @GET
+  @RolesAllowed("users")
+  @Path("{spaceId}/providers")
```

Comment:
In a future, we could replace spaceId by identityId, and have the same rest endpoint to get active space providers and active user providers.
We could change that in a future